apollo-router/src/executable.rs

```diff
@@ -21,8 +21,11 @@ static GLOBAL_ENV_FILTER: OnceCell<String> = OnceCell::new();
 
 /// Options for the router
 #[derive(Parser, Debug)]
-#[clap(global_setting(AppSettings::NoAutoVersion))]
-#[structopt(name = "router", about = "Apollo federation router")]
+#[clap(
+    name = "router",
+    about = "Apollo federation router",
+    global_setting(AppSettings::NoAutoVersion)
```

Comment:
![image](https://user-images.githubusercontent.com/9465678/168779297-9738a123-7399-40c1-a2f7-b71e66b60a3f.png)
what does this do? the docs aren't very explicit :D

Reply:
edit : nvm i wasnt looking at the latest docs https://docs.rs/clap/latest/clap/enum.AppSettings.html#variant.NoAutoVersion